man/nvim.1

```diff
@@ -1,4 +1,4 @@
-.Dd October 27, 2015
+cDd October 27, 2015
```

Comment:
`cDd` should be `.Dd`, I assume this was an accident. Also please update the date to today.